autoload/neomake/makers/python.vim

```diff
@@ -9,7 +9,7 @@ function! neomake#makers#python#EnabledMakers()
     if neomake#utils#Exists('flake8')
         call add(makers, 'flake8')
     else
-        call extend(makers, ['pep8', 'pyflakes'])
+        call extend(makers, ['pep8', 'frosted', 'pyflakes'])
     endif
     call add(makers, 'pylint')  " Last because it is the slowest
```

Comment:
I think we could change this line to:

```
call extend(makers, ['frosted', 'pylint'])  " Pylint is last because it is the slowest
```

And remove frosted above. The reason flake8 has that conditional craziness is because it has pep8 and pyflakes as dependencies, and adding them all into the list would result in pep8 and pyflakes running twice for no reason if flake8 was installed. Unless flake8 might run frosted (I'm guessing it won't since it relies on pyflakes), I think it can simply go in the list.